RELEASE.md

```diff
@@ -1,9 +1,9 @@
-## Release 2.5.0
+## Release 2.5.1
 
 Updated native SDK versions:
-* Android from `2.3.0` to [2.4.1](https://github.com/Microsoft/AppCenter-SDK-Android/releases/tag/2.4.1)
-* iOS from `2.4.0` to [2.5.0](https://github.com/Microsoft/AppCenter-SDK-Apple/releases/tag/2.5.0)
-* UWP from `2.1.0` to [2.5.0](https://github.com/Microsoft/AppCenter-SDK-DotNet/releases/tag/2.5.0)
+* Android from `2.4.1` to [2.5.0](https://github.com/Microsoft/AppCenter-SDK-Android/releases/tag/2.5.0)
```

Comment:
```suggestion
* Android from `2.3.0` to [2.5.0](https://github.com/Microsoft/AppCenter-SDK-Android/releases/tag/2.5.0)
```

Reply:
Because needs to be from last release we actually shipped.

Reply:
I've made changes, please check out